.github/workflows/ci.yml

```diff
@@ -105,17 +153,15 @@ jobs:
           node-version: 18.15
 
       - name: Install Node dependencies
-        run: npm install -g hardhat
+        run: npm install hardhat
 
       - name: Install Python dependencies
         run: |
-          pip3 install solc-select
-          pip3 install slither-analyzer
+          pip3 install --no-cache-dir solc-select slither-analyzer
```

Comment:
It's still uncertain whether we will integrate slither with medusa so can we change `slither-analyzer` to `crytic-compile`?